testData/solutions/win32.json5

```diff
@@ -675,56 +828,129 @@
                             "default": 1
                         }
                     },
-                    "HTML.MaxLineLength": {
+                    "HTML.IdentifySamePageLinks": {
                         "schema": {
-                            "title": "Max Line Length",
-                            "description": "The maximum number of characters which can appear on a line of a paragraph.",
-                            "type": "integer",
-                            "default": 150
+                            "title": "Identify Same Page Links",
+                            "description": "Whether to identify links that point to other places on the current page by saying \"same page link\".",
+                            "enum": [
+                                0,
+                                1
+                            ],
+                            "enumLabels": [
+                                "off",
+                                "on"
+                            ],
+                            "default": 1
                         }
                     },
-                    "HTML.PageRefreshFilter": {
+                    "HTML.LinkText": {
                         "schema": {
-                            "title": "Page Refresh Filter",
-                            "description": "How often, in milliseconds, to allow all refreshes.  Set to 0 to allow all refreshes without any delay.",
-                            "type": "integer",
-                            "default": 0
+                            "title": "",
+                            "description": "Select what to speak for an image.",
+                            "enum": [
+                                0,
+                                1,
+                                2,
+                                3,
+                                4
+                            ],
+                            "enumLabels": [
+                                "Title",
+                                "Screen Text",
+                                "Tool Tip",
+                                "Longest of Above",
+                                "The first attribute found in the order list"
+                            ],
+                            "default": 1
                         }
                     },
-                    "HTML.SayAllOnDocumentLoad": {
+                    "HTML.CustomTextLinkAttrs": {
                         "schema": {
-                            "title": "Read Web Pages Automatically When Loaded",
-                            "description": "Whether to automatically read web pages when they're first loaded.",
+                            "title": "Custom Text Link Attributes",
+                            "description": "Specify the order in which attributes of text links should be searched.",
+                            "type": "string",
+                            "default": "text|title|alt|href"
```

Comment:
Another variation on pipe-delimited strings, in this case defining ordering.  I can see that this one would be prohibitively difficult to enumerate.  I'm thinking this is a good use case for an aliased setting, i.e. the user-facing setting is an array of strings and we join it with a transform to make the settings handler facing setting.

Reply:
I believe most/all of these have been commented out now.